server/controllers/locations/getSuperUserRecords.js

```diff
@@ -0,0 +1,29 @@
+const boom = require("@hapi/boom");
+
+const connectToDatabase = require("../../database/dbConnection");
+const getSuperUserUsers = require("../../database/queries/user/getSuperUserUsers");
+const getRentalRecordsByIds = require("../../database/queries/rentalRecord/getRentalRecordsByIds");
+const isValidMongoID = require("../../utils/isValidMongoID");
+
+module.exports = async function getLocations(req, res, next) {
+  const { id: userId } = req.params;
+
+  if (!isValidMongoID(userId)) {
+    return next(boom.badData("unvalid user ID"));
+  }
+
+  // check the user role and return 403 if not authorized
+
+  try {
+    await connectToDatabase();
+    const users = await getSuperUserUsers(userId);

```

Comment:
Could this query look up rental records where the superuser's ID is in the verifiedBy field?

Reply:
Sorry I mean `reviewedBy`

Reply:
Actually, ignore me. I read the original issue and I agree with how you've queried for the data